src/main/resources/templates/branches-list.html

```diff
@@ -11,7 +11,7 @@
     <p>Branches:</p>
     <ol>
         <li th:id="${branch.key.id}" th:each="branch : ${branchList}">
-            <p th:text="${branch.key.city}"></p>
```

Comment:
To chyba może zostać, warto podać  informację zarówno o nazwie, jak i mieście. Tylko można przerzucić niżej, nad street.